app/src/main/java/kr/co/nottodo/presentation/mypage/view/MyPageInformationFragment.kt

```diff
@@ -99,6 +115,36 @@ class MyPageInformationFragment :
         }
     }
 
+    private fun gotoSystemSetting(isSystemSwitch: Boolean, isAppCheck: Boolean) {
```

Comment:
이 부분 파라미터를 nullable하게 한 후 내부에서 null 처리를 하는 것이 좀 더 확장성 있는 함수라고 생각됩니다! 추가적으로 앱 내의 알림 허용 스위치가 사라질 것 같다고 하네요 ..!

Reply:
추가적으로 기존의 navigate라는 워딩을 사용했으니 이것도 통일해보는 것 어떨까요 !?
```suggestion
    private fun navigateToSystemSetting(isSystemSwitch: Boolean, isAppCheck: Boolean) {
```